Makefile

```diff
@@ -228,8 +229,8 @@ images: $(IMAGES_BUILD)
 
 ### Typechecking ###
 # grab the bin from https://github.com/facebook/flow/releases
-flow: build/js/all.js
-	grep -Ev '/\*:: *(ex|im)port ' $< | flow check-contents
+flow: $(JSFILES)
+	grep -Ev '/\*:: *(ex|im)port ' $< | npm run flow check-contents
```

Comment:
Can the `grep` be removed now?

Reply:
Thought of that, too. This `grep` strips all lines containing the regex. When the comment has a line break, it keeps everything but the first line (not intended).

I don't know why this was introduced in the first place.

Reply:
Don't see an obvious reason to have it.

Reply:
> When the comment has a line break, it keeps everything but the first line (not intended).

Yeah this seems extra bad. Probably breaks the JS syntax wise?

> I don't know why this was introduced in the first place.
 
I think Flow uses the export lines and they would have worked except for the cylic dependencies. So, @unhammer just effectively commented them out before passing to flow in anticipation of eventually fixing the cyclic dependencies and making it so flow would actually use the export/imports. So, it should be removable now.